[2024-01-04T12:42:12.449+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: online_retail.up_data_to_gcs manual__2024-01-04T12:14:24.430451+00:00 [queued]>
[2024-01-04T12:42:12.475+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: online_retail.up_data_to_gcs manual__2024-01-04T12:14:24.430451+00:00 [queued]>
[2024-01-04T12:42:12.476+0000] {taskinstance.py:1361} INFO - Starting attempt 3 of 3
[2024-01-04T12:42:12.506+0000] {taskinstance.py:1382} INFO - Executing <Task(LocalFilesystemToGCSOperator): up_data_to_gcs> on 2024-01-04 12:14:24.430451+00:00
[2024-01-04T12:42:12.518+0000] {standard_task_runner.py:57} INFO - Started process 558 to run task
[2024-01-04T12:42:12.524+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'online_retail', 'up_data_to_gcs', 'manual__2024-01-04T12:14:24.430451+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmptkk92i_k']
[2024-01-04T12:42:12.526+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask up_data_to_gcs
[2024-01-04T12:42:12.623+0000] {task_command.py:416} INFO - Running <TaskInstance: online_retail.up_data_to_gcs manual__2024-01-04T12:14:24.430451+00:00 [running]> on host e552979bdd1c
[2024-01-04T12:42:12.741+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='online_retail' AIRFLOW_CTX_TASK_ID='up_data_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-01-04T12:14:24.430451+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-04T12:14:24.430451+00:00'
[2024-01-04T12:42:12.757+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-04T12:42:12.763+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-04T12:43:05.124+0000] {gcs.py:562} INFO - File /opt/***/dags/includeonline_retail.csv uploaded to online_retail.csv in online-retail-data bucket
[2024-01-04T12:43:05.150+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=online_retail, task_id=up_data_to_gcs, execution_date=20240104T121424, start_date=20240104T124212, end_date=20240104T124305
[2024-01-04T12:43:05.228+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-04T12:43:05.249+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
